lib/expedia.js

```diff
@@ -86,8 +86,8 @@ module.exports = function (options, sendAsREST) {
                 return {
                     HotelListResponse: {
                         HotelList: {
-                            '@size': 0,
-                            '@activePropertyCount': 0,
+                            '@size': '0',
```

Comment:
Doesn’t it make more sense as a number? It’s a record count

Reply:
Or we can parse all the numbers during transformation, but I can't say they are always send parsable strings.

Reply:
Doesn’t it make sense to parse things into what we want, not the other way round?

Reply:
If they don't send it parseable, that's the BE problem, we cannot pass this to FE, unless we mention in the contract that it's a string. Also if we are not manipulating this number or not using this number then we can pass it as a string. Currently FE is not using these numbers at all tho.

Reply:
parse it in search module, I'm closing this without merging